src/mbgl/util/offscreen_texture.cpp

```diff
@@ -40,10 +43,11 @@ class OffscreenTexture::Impl {
     void bindRenderbuffers(gl::TextureUnit unit) {
         if (!framebuffer) {
             texture = context.createTexture(size, gl::TextureFormat::RGBA, unit);
-            gl::Renderbuffer<gl::RenderbufferType::DepthComponent> depthTarget = context.createRenderbuffer<gl::RenderbufferType::DepthComponent>(size);
-            framebuffer = context.createFramebuffer(*texture, depthTarget);
-
+            gl::Renderbuffer<gl::RenderbufferType::DepthComponent> depth =
+                context.createRenderbuffer<gl::RenderbufferType::DepthComponent>(size);
+            framebuffer = context.createFramebuffer(*texture, depth);
         } else {
+            context.bindTexture(*texture, unit);
```

Comment:
🤕 
Hm. This didn't work without this line — the map was just fully black. Upon further investigation, it's because it needed [`context.textures[unit] = texture`](https://github.com/mapbox/mapbox-gl-native/blob/dd353a60d1656f13351e8caa69e902b7f3111a02/src/mbgl/gl/context.cpp#L460). I think actually this is because something somewhere reset the texture at unit 1. I did just realize I'm able to avoid needing this line by moving the extrusion texture to unit 2. (We should refactor this down the line to automatically manage texture units…)